en/intro_to_command_line/README.md

```diff
@@ -146,6 +146,24 @@ You'll probably see something similar on your machine. Once you open the command
 
 ---
 
+### Learn more about a command
+
+Many commands you can type at the command prompt have built-in help that you can display and read! For example, to learn more about the Current directory command:
+
+<!--sec data-title="Command help: OS X and Linux" data-id="OSX_Linux_pwd" data-collapse=true ces-->
+
+OS X and Linux have a `man` command, which gives you help on commands. Try `man pwd` and see what it says, or put `man` before other commands to see their help.
```

Comment:
If the man page doesn't fit in the command line window, `man` will automatically invoke a pager (usually `less`) to display the content. It should probably be mentioned how to navigate in there (arrow keys, page-up/page-down keys) and how to exit the pager (`q` for "quit").

Reply:
Good idea! I'll update the text...